README.md

```diff
@@ -1,67 +1,71 @@
-# Author-Endorser-Wizards
-
-Python program to walk through the author and endorser workflow to create schemas and cred defs 
-
-## Docker Instructions
-
-### Running the Endorser
-once you have cloned the repo,
-
-* docker build -t endorser-wizard -f endorser-docker-file . (the dot is important, as it signifys the current directory)
-* docker run --rm -it -v \<Path to git repository\>/author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>/wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>/pool:/root/.indy_client/pool:z endorser-wizard 
-  * \(If on windows run: docker run --rm -it -v \<Path to git repository\>\author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>\wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>\pool:/root/.indy_client/pool:z endorser-wizard\)
-### Running the Author
-* docker build -t author-wizard -f author-docker-file . (the dot is important, as it signifys the current directory)
-* docker run --rm -it -v \<Path to git repository\>/author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>/wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>/pool:/root/.indy_client/pool:z author-wizard
-  * \(If on windows run: docker run --rm -it -v \<Path to git repository\>\author-endorser-wizards:/root/author-endorser-wizards:z -v \<Path to .indy_client\>\wallet:/root/.indy_client/wallet:z -v \<Path to .indy_client\>\pool:/root/.indy_client/pool:z author-wizard
-
-
-## Alternate Setup Instructions
-If your not using Docker, here are the individual setup and running instructions you can follow to run the author and endorser wizards
-
-### Linux 18.04
-
- On a clean install of Ubuntu 18.04 the following works to install and run the Appropriate Wizard.  Please adjust if appropriate for your environment.
-* sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
-* sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb bionic stable"
-* sudo apt-get update
-* sudo apt-get upgrade
-* sudo apt-get install -y libindy
-* sudo apt install python3.8 -y
-* sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1
-* sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2
-* sudo apt install python3-pip git
-* pip3 install --upgrade pip
-* pip3 install aiohttp base58 indy Python3-indy asyncio
+[**Docker Instructions**](#docker)  
```

Comment:
will it look nicer to move the title to the top, and then have the TOC?

Reply:
Agreed, let's move the title and one line introduction above the TOC.

Reply:
Agreed, let's move the title and one line introduction above the TOC.